classes/class-kp-assets.php

```diff
@@ -97,6 +97,7 @@ private function get_checkout_params( $settings ) {
 			'submit_order'           => WC_AJAX::get_endpoint( 'checkout' ),
 			// Params.
 			'testmode'               => $settings['testmode'] ?? 'no',
+			'debug'                  => defined( 'WP_DEBUG' ) && WP_DEBUG ? 'yes' : 'no',
```

Comment:
There is a different definition for script debug that we can use instead. `SCRIPT_DEBUG`. You can find that [here](https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/#script_debug).